packages/p2p/src/stores/advertiser-page-store.js

```diff
@@ -107,13 +110,15 @@ export default class AdvertiserPageStore extends BaseStore {
                     ? { local_currency: buy_sell_store.selected_local_currency }
                     : {}),
             }).then(response => {
-                if (response.error) {
-                    this.setErrorMessage(response.error);
-                } else {
-                    const { list } = response.p2p_advert_list;
-
-                    this.setAdverts(list);
-                    this.setHasMoreAdvertsToLoad(list.length >= general_store.list_item_limit);
+                if (response) {
+                    if (response.error) {
+                        this.setErrorMessage(response.error);
+                    } else {
+                        const { list } = response.p2p_advert_list;
+
+                        this.setAdverts(list);
+                        this.setHasMoreAdvertsToLoad(list.length >= general_store.list_item_limit);
+                    }
```

Comment:
this was the error that showed:

<img width="414" alt="Screenshot 2023-08-21 at 11 19 02 AM" src="https://github.com/ameerul-deriv/deriv-app/assets/103412909/f84387c4-8d15-49ba-80f2-5dfb6835cb8d">